packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -272,9 +268,12 @@ export class AccountTrackerController extends StaticIntervalPollingControllerV1<
 
       const accountsForChain = { ...accountsByChainId[chainId] };
       for (const address of accountsToUpdate) {
-        accountsForChain[address] = {
-          balance: BNToHex(await this.getBalanceFromChain(address, ethQuery)),
```

Comment:
no need for BNToHex in this context as getBalanceFromChain returns balance in hex format